packages/block-editor/src/components/block-list-appender/index.js

```diff
@@ -51,26 +54,24 @@ function BlockListAppender( {
 		);
 	}
 
-	// IgnoreNestedEvents is used to treat interactions within the appender as
-	// subject to the same conditions as those which occur within nested blocks.
-	// Notably, this effectively prevents event bubbling to block ancestors
-	// which can otherwise interfere with the intended behavior of the appender
-	// (e.g. focus handler on the ancestor block).
-	//
-	// A `tabIndex` is used on the wrapping `div` element in order to force a
-	// focus event to occur when an appender `button` element is clicked. In
-	// some browsers (Firefox, Safari), button clicks do not emit a focus event,
-	// which could cause this event to propagate unexpectedly. The `tabIndex`
-	// ensures that the interaction is captured as a focus, without also adding
-	// an extra tab stop.
-	//
-	// See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#Clicking_and_focus
 	return (
-		<IgnoreNestedEvents childHandledEvents={ [ 'onFocus', 'onClick', 'onKeyDown' ] }>
-			<div tabIndex={ -1 } className="block-list-appender">
-				{ appender }
-			</div>
-		</IgnoreNestedEvents>
+		<div
+			// A `tabIndex` is used on the wrapping `div` element in order to
+			// force a focus event to occur when an appender `button` element
+			// is clicked. In some browsers (Firefox, Safari), button clicks do
+			// not emit a focus event, which could cause this event to propagate
+			// unexpectedly. The `tabIndex` ensures that the interaction is
+			// captured as a focus, without also adding an extra tab stop.
+			//
+			// See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#Clicking_and_focus
+			tabIndex={ -1 }
+			// Prevent the block from being selected when the appender is
+			// clicked.
```

Comment:
I was actually surprised to find that in the latest version of the plugin / core version, the behavior of clicking on the inserter in the column block is such that it requires a second click to open. I don't think it's always been like this, since I distinctly remember trying to require only a single click. One potential difference now is that, while it does only require a single click, it doesn't actually select the block. I don't particularly see this as being problematic, and in some ways it's actually an ideal behavior (since the user's intent is to insert, not necessarily to select the block).